doc/sphinx/source/api/CeedQFunction.rst

```diff
@@ -11,3 +11,12 @@ Resolution/space-independent weak forms and quadrature-based operations
    :path: ../../../../xml
    :content-only:
    :members:
+
+Macros
+--------------------------------------
+
+.. doxygendefine:: CEED_QFUNCTION
```

Comment:
Thanks for adding these macros to the documentation @jeremylt. Do you think this PR could also fit the documentation for the QFunction Gallery that we were discussing on Slack?

Reply:
Seems out of scope; I'd rather merge this to release #474 and anything else that might depend on it.

Reply:
Agreed. Maybe a small separate PR?